source/Engine.cpp

```diff
@@ -1625,8 +1630,17 @@ void Engine::HandleKeyboardInputs()
 	else
 		activeCommands.Clear(Command::WAIT);
 	
-	// Transfer all newly pressed, unhandled keys to active commands.
-	activeCommands |= keyDown;
+	// Translate shift+BACK to a command to a STOP command to stop all movement of the flagship.
+	// Translation is done here to allow the autoPilot (which will execute the STOP-command) to
+	// act on a single STOP command instead of the shift+BACK modifier).
+	if(keyHeld.Has(Command::BACK) && keyHeld.Has(Command::SHIFT))
```

Comment:
Alternately, we could keep the current `| manueveringCommands);` bit, and then use `activeCommands.Clear(Command::BACK)` if BACK needs to be replaced with STOP. That might result in clearer logic.

Reply:
Yes, this is the alternative I took.